src/main/java/use_cases/add_edit_expenses_use_case/ExpenseUCI.java

```diff
@@ -190,22 +190,31 @@ private boolean changeInRecurringInfo() {
             try{findExpense(expenseID.getSession().getRecurData(), expenseID.getOldExpense());
                 return true;
             }catch(NoSuchElementException e) {
-                return false;}}}
+                return false;}}
+    }
 
     /**
-     * Edits selected Expense prepared with necessary edit info from expenseID
-     * @param expenseID relevant expenseID
-     * @param valueDouble new expense value for edit
-     * @param selectedExpense Expense wish to edit
-     * @param selectedCategory Category the selected Expense belongs to
+     * Sets the information needed to create an ExpenseCreatorInputData to call createMonthObject method in ExpenseFactory
+     * @return ExpenseCreatorInputData MonthObjectFactoryInputData Object specifically used in ExpenseFactory
+     * for the createMonthObject method.
      */
-    private void setEditInfo(ExpenseID expenseID, double valueDouble, Expense selectedExpense, Category selectedCategory){
-        selectedExpense.setName(expenseID.getName());
-        expenseID.setValue(valueDouble);
-        selectedExpense.setValue(valueDouble);
-        selectedExpense.setCategory(selectedCategory);}
-
-}
-
-
+    private MonthObjectFactoryInputData setExpenseCreatorInfo(ExpenseID expenseIDAdd){
```

Comment:
Since this method is making a Factory Input Data, why not use the words Input Data instead of info for the method name. I think that could be clearer

Reply:
Or Factory Input Data if you don't want it to be confused for the UCI input data. Something likesetExpenseCreatorFactoryInputData. Yes the name is very long but to be honest I think that would be clearer for someone reading the code for the first time. I had to keep shuffling up and down while doing my review since I didn't understand just from the name what these methods did